test/integration/local/routing/update-requests-awaiting-debt-data.test.js

```diff
@@ -0,0 +1,43 @@
+const { resetDatabase, closeDatabaseConnection, savePaymentRequest } = require('../../../helpers')
+
+jest.mock('../../../../app/routing/prepare-for-reprocessing')
+const { prepareForReprocessing: mockPrepareForReprocessing } = require('../../../../app/routing/prepare-for-reprocessing')
+
+const { RECOVERY_DATE } = require('../../../mocks/values/recovery-date')
+
+const { ADMINISTRATIVE } = require('../../../../app/constants/debt-types')
+
+const { updateRequestsAwaitingDebtData } = require('../../../../app/routing/update-requests-awaiting-debt-data')
+
+let paymentRequest
+
+describe('update requests awaiting debt data', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks()
+    paymentRequest = JSON.parse(JSON.stringify(require('../../../mocks/payment-requests/payment-request')))
+    await resetDatabase()
```

Comment:
Does this await need to be defined here? Or can it be defined below the jest.clearAllMocks?

Reply:
moved